pkg/collector/runner/expvars/expvars_test.go

```diff
@@ -398,3 +400,41 @@ func TestExpvarsToplevelKeys(t *testing.T) {
 		changeAndAssertExpvarValue(t, keyName, setter, getters[keyName], 3, 4)
 	}
 }
+
+func TestGetCheckStatsRace(t *testing.T) {
+	Reset()
```

Comment:
Should we use the test help function `setUp()`?

Reply:
I didn't really understand the point of that function, it's purely just `Reset()`, I thought it was clearer to call it explicitly
Not all tests even call it by the way 🙃

Reply:
> I thought it was clearer to call it explicitly

It does makes sense. We could improve the rest of the test by adding the same cleanup code you added nd remove the `setup()` function all together 

```go
Reset()
t.Cleanup(Reset)
```

That way we standardize the test 😄